src/phpDocumentor/Transformer/Writer/Twig/Extension.php

```diff
@@ -147,14 +144,14 @@ static function (
             ),
             'sort' => new TwigFilter(
                 'sort_*',
-                static function ($direction, $collection) {
+                static function (string $direction, $collection) {
```

Comment:
can we add the return type on this closure as well?

Reply:
Unfortunately not without looking into this some more; it seems that $collection may not always be a collection. I stayed on the safe side with this one because I didn't dare to :)